sdk/storage/storage-internal-avro/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+# Release History
+
+## 1.0.0 (Unreleased)
```

Comment:
storage-internal-avro is not a published package? Since not, we don't need this changelog file.

Reply:
Let's take it as a package except it's not published. Added it to `rush.json` so that we can update dependencies and run tests independently for the avro package.
Basically I am following the routine of the test-utils-recorder:
https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/test-utils/recorder/CHANGELOG.md

Reply:
I'd like to keep the CHANGELOG.md as it is.
Since it's more like a package that's not published, (added it to the rush.json as utility: https://github.com/Azure/azure-sdk-for-js/blob/c94cc0ac61fef067f413f49d784fe1f505e4b9f0/rush.json#L490), I am basically following the routine of test-util-recorder: https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/test-utils/recorder/CHANGELOG.md